servicemodels/e2sm_ccc/pdubuilder/cc-action-definition_test.go

```diff
@@ -0,0 +1,61 @@
+package pdubuilder
```

Comment:
Add license/copyright header in all of the new files
```suggestion
// SPDX-License-Identifier: Apache-2.0
// Copyright 2023-present Intel Corporation

package pdubuilder
```